posthog/temporal/tests/batch_exports/test_s3_batch_export_workflow.py

```diff
@@ -972,7 +980,7 @@ async def test_s3_export_workflow_with_minio_bucket(
     "data_interval_start",
     # This is hardcoded relative to the `data_interval_end` used in all or most tests, since that's also
     # passed to `generate_test_data` to determine the timestamp for the generated data.
-    [dt.datetime(2023, 4, 24, 15, 0, 0, tzinfo=dt.UTC)],
+    [dt.datetime.now(tz=dt.UTC).replace(hour=0, minute=0, second=0, microsecond=0) - dt.timedelta(hours=24)],
```

Comment:
Same as previous comment, why not `- dt.timedelta(hours=1)`?

Reply:
as above